frontend/schema.faker.graphql

```diff
@@ -0,0 +1,81 @@
+# This is sample SDL schema for GraphQL Faker.
```

Comment:
in `schemes` oder `graphql` subfolder packen (oder irgendein anderes passendes naming....)